lib/removeUnusedCameras.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+module.exports = removeUnusedCameras;
+
+function removeUnusedCameras(gltf, stats) {
+    var usedCameraIds = {};
+    var nodes = gltf.nodes;
+
+    // Build hash of used cameras by iterating through nodes
+    if (defined(nodes)) {
+        for (var nodeId in nodes) {
+            if (nodes.hasOwnProperty(nodeId)) {
+                if(nodes[nodeId].hasOwnProperty("camera")){
```

Comment:
This should be
```
if (defined(nodes[nodeId].camera)) {
```

Also, as far as style for spacing and all things code, let's following the [Cesium Coding Guide](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/CodingGuide/README.md) - it is a long-ish read and not everything applies since this is Node.js (compared to client-side JavaScript), but we have established a number of best practices after years of experience.  Check it out.